mix.exs

```diff
@@ -28,8 +28,8 @@ defmodule TimexEcto.Mixfile do
   #
   # Type `mix help deps` for more examples and options
   defp deps do
-    [{:timex, "~> 3.0"},
-     {:ecto, "~> 2.1.0"},
+    [{:timex, "~> 3.1"},
```

Comment:
@bitwalker Version here seems to be at least 3.1.15, in which `to_datetime({{y,m,d},{h,mm,s,_us}} = dt, timezone)` was supported the first time.